app/containers/LeftMenu/Documentation/ItemMenu.tsx

```diff
@@ -189,57 +238,22 @@ const ItemMenu: React.FC<DocumentationMenuProps> = ({
               }}
             >
               <Dropdown
-                trigger={
+                id="itemMenu_dropdown_id"
```

Comment:
check if this prop is needed and delete if not

Reply:
yes, this is a required prop